schema/network.yaml

```diff
@@ -68,6 +68,12 @@ definitions:
         type: string
         readOnly: true
         description: Indicate the interface name.
+      status:
+        type: boolean
+        description: Connection status of the interface, true for connected.
+      wan:
+        type: boolean
+        description: Indicate the interface worked as WAN or not (LAN).
```

Comment:
I think we could add another endpoint to let other bundles able to modify route priority. In this case, wifi bundle should remove the interface which is in ap mode from routing list. Otherwise, route bundle restart will loose these information.